src/openforms/variables/migrations/0004_migrate_query_params.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.18 on 2023-03-13 15:38
+
+import json
+
+from django.db import migrations
+
+
+def migrate_query_params_to_jsonfield(apps, _):
+    ServiceFetchConfiguration = apps.get_model("variables", "ServiceFetchConfiguration")
+
+    for config in ServiceFetchConfiguration.objects.all():
+        try:
+            config._query_params = json.loads(config.query_params)
+        except json.decoder.JSONDecodeError:
+            config._query_params = {}
+        config.save()
+
+
+def migrate_query_params_to_textfield(apps, _):
+    ServiceFetchConfiguration = apps.get_model("variables", "ServiceFetchConfiguration")
+
+    for config in ServiceFetchConfiguration.objects.all():
+        config.query_params = json.dumps(config._query_params)
+        config.save()
```

Comment:
this should do the `urlencode` bit instead, I believe, but may not escape the curly braces syntax.